sycl/include/CL/sycl/detail/image_accessor_util.hpp

```diff
@@ -196,9 +196,6 @@ vec<T, 4> readPixel(T *Ptr, const image_channel_order ChannelOrder,
     Pixel.y() = Ptr[2]; // g
     Pixel.x() = Ptr[3]; // r
     break;
-  default:
-    throw cl::sycl::invalid_parameter_error("Unhandled image channel order",
-                                            PI_INVALID_VALUE);
```

Comment:
Why was this removed? Can we add something else? Will an assert also give a warning?

Reply:
@garimagu that code handles all cases from an enum. It'd never reach `default:` branch anyway.

Reply:
Ok.. So you get this warning because of this "-Wcovered-switch-default: remove unreachable default clauses" warning?

Reply:
Right, this is a new warning, introduced in recent GCC versions, that's why it's not hitting us in CI.

Reply:
@garimagu please, unresolve this conversations if you still have questions.